### PR TITLE
Create greasy-molue.html

### PR DESCRIPTION
**Pull request by greasymolue**
Hope you like it.

### CODE REVIEW

**Comment by greasymolue** [comment]
Fixes issue #9

File: greasy-molue.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<h1>Greasy</h1>
+<u1>
+<li>Favorite foods: french fries, chicken wings and cheese curds</li>
+<li>Back story: little brother had a friend who went by the greasy moolie, which became an inside joke. He did not know how to spell moolie, and instead wrote "greasy molue" when hacking our mother's cellphone welcome screen. When I created this account last year, I knew nobody else would have this alias. I also have the same alias on XDA-developers.</li>
```
Comment (mrmorris) [inline greasy-molue.html]: I laughed. I cried.

**Comment by greasymolue** [comment]
Made it shorter.

**Comment by tpesce** [comment]
:+1: looks good!

**Pull request merged**